tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs

```diff
@@ -180,13 +182,22 @@ async Task ImportVstsReleasesDataAsync(ReleaseManagement releaseManagement, stri
                             {
                                 UpsertVstsReleaseDeploymentToDb(sqlConnection, releaseEnvironment.Id, deployment);
 
-                                foreach(IoTEdgePipelineTask pipelineTask in deployment.Tasks.Where(x => IsTestTask(x)))
+                                const string testTaskPrefix = "Test:";
```

Comment:
My understanding is that you are adding a prefix to all test tasks in the test yaml then stripping the substring out when loading the tasks into the db. 

Do we have the changes in master yet to change the task names, or is it a hardcoded yaml in the pipeline?

Reply:
Old e2e tests is not in yaml; just in vsts pipeline. :(